api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -127,19 +127,26 @@ public void setUserService(UserService userService) {
   }
 
   @Override
-  public ResponseEntity<ClusterListResponse> listClusters() {
+  public ResponseEntity<ClusterListResponse> listClusters(String projectName) {
     User user = this.userProvider.get();
+    // TODO as part of Billing Project epic RW-1205: enable status checking for arbitrary project
+    // until then: continue to check the user's free tier project status
+    // because that is the only project we'll pass in here
+    if (!user.getFreeTierBillingProjectName().equals(projectName)) {
+      throw new FailedPreconditionException(
+          "NOT YET IMPLEMENTED: Cannot list clusters using arbitrary project name");
+    }
     if (user.getFreeTierBillingProjectStatusEnum() != BillingProjectStatus.READY) {
       throw new FailedPreconditionException(
           "User billing project is not yet initialized, cannot list/create clusters");
     }
-    String project = user.getFreeTierBillingProjectName();
     org.pmiops.workbench.notebooks.model.Cluster fcCluster;
     try {
-      fcCluster = this.leonardoNotebooksClient.getCluster(project, LeonardoNotebooksClient.DEFAULT_CLUSTER_NAME);
+      fcCluster = this.leonardoNotebooksClient.getCluster(projectName,
```

Comment:
These lines look like auto-format churn. I don't recall for sure, but maybe our team uses 100 char width for Java files and your editor is set to 80?

TBH, I don't usually worry too much about formatting issues like this. We should eventually set up a clang-format commit-hook which frees all of us from ever having to think about formatting again.